README.md

```diff
@@ -1,11 +1,131 @@
 # irctest
 
-This is the integration test suite for Oragono, ultimately derived from [ProgVal/irctest](https://github.com/ProgVal/irctest), which is a general-purpose IRC protocol compatibility testing suite.
+This project aims at testing interoperability of software using the
+IRC protocol, by running them against test suites and making different
+software communicate with each other.
 
-Some of these tests may be applicable to other projects (we attempt to mark the tests that are only applicable to Oragono).
+It is very young and does not contain a lot of test cases yet.
 
-This suite needs more test cases. Contributions are welcome and are a great way to help the Oragono project!
+## The big picture
 
-## Running Tests
+This project contains:
+
+* IRC protocol test cases
+* small wrappers around existing software to run tests on them
+
+Wrappers run software in temporary directories, so running `irctest` should
+have no side effect, with [the exception of Sopel](https://github.com/sopel-irc/sopel/issues/946).
+
+## Prerequisites
+
+Install irctest and dependencies:
+
+```
+git clone https://github.com/ProgVal/irctest.git
+cd irctest
+pip3 install --user -r requirements.txt pyxmpp2-scram
+python3 setup.py install --user
+```
+
+Add `~/.local/bin/` to your `PATH` if it is not.
```

Comment:
Similarly, none of these other install/preparatory steps is necessary in the oragono/irctest fork.

Reply:
What? How does irctest find oragono then?

Reply:
Ah, it's just on my `$PATH` (I use `go install` when I build it).

How come `~/.local/bin`, though? The build instructions for oragono later in this README say to put it in `/tmp`.

Reply:
Oh wait no, that's not for ora. I don't know why I wrote this. I'll remove it.

Reply:
Oh I get it now. `~/.local/bin` is for every software **but** oragono, because oragono doesn't have a `make install` or equivalent AFAIK

Reply:
I mean, it does not have a `make install` with a configurable prefix

Reply:
`go install` conventionally installs the binary into `${GOPATH}/~/bin`, typically `~/go/bin`.

Could we add a note explaining what's going on, e.g., "ensure the correct version of the ircd you want to test is on your `$PATH`"?

Reply:
okay let's add `~/go/bin` to the path too.

People who know how PATH works will skip that section, and it makes life easier for everyone else